gwpy/table/gravityspy.py

```diff
@@ -177,3 +189,44 @@ def _download_single_image(url):
             if isinstance(x, Exception):
                 x.args = ('Failed to read %s: %s' % (f, str(x)),)
                 raise x
+
+    @classmethod
+    def search(cls, uniqueID, howmany=10):
+        """If table contains Gravity Spy triggers `EventTable`
+
+        Parameters
+        ----------
+        uniqueID : `str`,
+            This is the unique 10 character hash that identifies
+            a Gravity Spy Image
+
+        howmany : `int`, optional, default: 10
+            number of similar images you would like
+
+        kwargs:
+
+        Returns
+        -------
+        Folder containing omega scans sorted by label
+        """
+        # back to pandas
+        try:
+            import pandas
+        except ImportError as exc:
+            exc.args = ('pandas is required to download triggers',)
+            raise
+
+        # Need to build the url call for the restful API
+        url = 'https://gravityspytools.ciera.northwestern.edu' + \
+            '/search/similarity_search_restful_API/?'
+        # The first varaible username for the url is unimportant
+        # and can permantly be 'sbc538'
+        username = 'sbc538'
+        url = url + 'username={0}&'.format(username)
+        url = url + 'howmany={0}&'.format(howmany)
+        # zooID can also be permanantly set to zero.
+        zooID = '0'
+        url = url + 'zooid={0}&'.format(zooID)
+        url = url + 'imageid={0}'.format(uniqueID)
+
+        return GravitySpyTable.from_pandas(pandas.read_json(url))
```

Comment:
@scottcoughlin2014, do we really need to use `pandas` for this construction?

Reply:
:( But it was so super fun! Alright give me a sec I am sure astropy can probably do it...but maybe not? :)

Reply:
```
from astropy.utils.data import get_readable_fileobj

...

        with get_readable_fileobj(url) as f:
            return GravitySpyTable(json.load(f))
```